src/policy/compiler.rs

```diff
@@ -164,38 +169,64 @@ impl Property for CompilerExtData {
             branch_prob: None,
             sat_cost: f64::MAX,
             dissat_cost: Some(0.0),
+            sig_type: PhantomData,
         }
     }
 
     fn from_pk_k() -> Self {
         CompilerExtData {
             branch_prob: None,
-            sat_cost: 73.0,
+            sat_cost: match Ctx::sig_type() {
+                SigType::Ecdsa => 73.,
+                SigType::Schnorr => 64.,
+            },
             dissat_cost: Some(1.0),
+            sig_type: PhantomData,
         }
     }
 
     fn from_pk_h() -> Self {
         CompilerExtData {
             branch_prob: None,
-            sat_cost: 73.0 + 34.0,
-            dissat_cost: Some(1.0 + 34.0),
+            sat_cost: match Ctx::sig_type() {
+                SigType::Ecdsa => 73.0 + 34.0,
+                SigType::Schnorr => 64. + 32.,
```

Comment:
nit: We should be consistent in using `.` and `.0`

Reply:
Addressed in 2565f5e, went for `.0` .